src/tools/wasm-fuzz-lattices.cpp

```diff
@@ -261,6 +267,36 @@ RandomFullLattice::Element RandomFullLattice::makeElement() const noexcept {
   if (std::get_if<UInt32>(lattice.get())) {
     return ElementImpl{rand.upToSquared(33)};
   }
+  if (std::get_if<ValType>(lattice.get())) {
+    Type type;
+    switch (rand.upTo(8)) {
+      case 0:
+        type = Type::unreachable;
+        break;
+      case 1:
+        type = Type::none;
+        break;
+      case 2:
+        type = Type::i32;
+        break;
+      case 3:
+        type = Type::f32;
+        break;
```

Comment:
Why not i64, f64?

Reply:
I don't think adding them (or v128, or other reference types) would add much, since their relationships to other types isn't any different than what is already covered here.

Reply:
Fair enough. Comment might be worthwhile as it's an obvious question for a reader of the code to have I think.

Reply:
Will do 👍